norfair/video.py

```diff
@@ -234,6 +238,8 @@ def __init__(self, input_path, save_path=".", information_file=None):
         self.input_path = input_path
         self.frame_number = 1
         self.video = cv2.VideoWriter(video_path, fourcc, fps, image_size)  # Video file
+        self.imExt = information_file.search("imExt")
+        self.imDir = information_file.search("imDir")
```

Comment:
Python code style states that objects should be named using 'lower_case_with_underscores' and classes should be named using 'CamelCase'. Also, it is better to use more descriptive names, unless they are way too long, which is not the case here.

Therefore, if I am understanding what these variables are correctly, I would rename them to `self.image_extension` and `self.image_directory`.